test/functional/commands/find/by-image-e2e-specs.js

```diff
@@ -13,14 +13,15 @@ const START_IMG = path.resolve(__dirname, '..', '..', 'assets', 'start-button.pn
 const STOP_IMG = path.resolve(__dirname, '..', '..', 'assets', 'stop-button.png');
 const SQUARES_IMG = path.resolve(__dirname, '..', '..', 'assets', 'checkered-squares.png');
 
-describe('Find - Image @skip-ci', function () {
+describe('Find - Image', function () {
   let driver;
 
   before(async function () {
-    driver = await initSession({
-      ...APIDEMOS_CAPS,
-      appActivity: '.view.ChronometerDemo'
+    this.skip();
```

Comment:
These tests failed due to `strategy "-image" is invalid` error. I wonder if UIA2 driver supports image strategy. 
https://github.com/appium/appium-uiautomator2-driver/blob/935a4e89de48942bb8cb8d9794ac0154ccec080f/lib/driver.js#L156-L161

Reply:
The CI needs to install https://www.npmjs.com/package/@appium/images-plugin.
The skip is sufficient. Could you leave TODO as to install the images plugin?

Reply:
Thanks! done
https://github.com/appium/appium-uiautomator2-driver/pull/578/commits/2c8940ce62f6808d67e4834a45bdd35cc9c999d0